#! Param 1
mass_arm: [5,0,0,0,0,0,
           0,5,0,0,0,0,
           0,0,5,0,0,0,
           0,0,0,5,0,0,
           0,0,0,0,5,0,
           0,0,0,0,0,5]
damping_arm: [45,0,0,0,0,0,
              0,45,0,0,0,0,
              0,0,45,0,0,0,
              0,0,0,45,0,0,
              0,0,0,0,45,0,
              0,0,0,0,0,45]
#stiffness_coupling: [100,0,0,0,0,0,
#                     0,100,0,0,0,0,
#                     0,0,100,0,0,0,
#                     0,0,0,100,0,0,
#                     0,0,0,0,100,0,
#                     0,0,0,0,0,100]
stiffness_coupling: [0,0,0,0,0,0,
                     0,0,0,0,0,0,
                     0,0,0,0,0,0,
                     0,0,0,0,0,0,
                     0,0,0,0,0,0,
                     0,0,0,0,0,0]

# #! Param 2
# mass_arm: [10,0,0,0,0,0,
#            0,10,0,0,0,0,
#            0,0,10,0,0,0,
#            0,0,0,10,0,0,
#            0,0,0,0,10,0,
#            0,0,0,0,0,10]
# damping_arm: [80,0,0,0,0,0,
#               0,80,0,0,0,0,
#               0,0,80,0,0,0,
#               0,0,0,80,0,0,
#               0,0,0,0,80,0,
#               0,0,0,0,0,80]
# stiffness_coupling: [80,0,0,0,0,0,
#                      0,80,0,0,0,0,
#                      0,0,80,0,0,0,
#                      0,0,0,80,0,0,
#                      0,0,0,0,80,0,
#                      0,0,0,0,0,80]

base_link: "base_link"
end_link: "wrist_3_link"
# [position (x, y, z)  orientation (x, y, z, w)]
desired_pose:  [-0.4,0.0,0.50,       0.707, -0.707, 0.0, 0.0]

# desired_pose:  [0.5,0.0,0.50,       0.477, -0.480, 0.646, -0.353]
workspace_limits: [-0.50, 0.50, 0.25, 0.80, 0.30, 0.75]

arm_max_vel: 3.14
arm_max_acc: 3.14
platform_max_vel: 0.1
platform_max_acc: 0.2